docs/concepts/ipfs-implementations.md

```diff
@@ -1,47 +1,63 @@
+A comprehensive list of [IPFS implementations](./implementations.md) across different languages and use cases, from desktop applications to specialized libraries.
 
-You can learn more about the principles that define what an IPFS implementation is [here](./implementations.md).
+- [Desktop Implementations](#desktop-implementations) 
+- [Popular Node Implementations and Tools](#popular-node-implementations-and-tools)
+- [Content-Addressed Data](#content-addressed-data)
+- [Lite or Experimental](#lite-or-experimental)
+- [Inactive](#inactive))
 
+To propose additions or edits, [edit this page in GitHub](https://github.com/ipfs/ipfs-docs/edit/main/docs/concepts/ipfs-implementations.md) or [open an issue](https://github.com/ipfs/ipfs-docs/issues/new?assignees=&labels=need%2Ftriage&template=open_an_issue.md&title=IPFS%20Implementations).
 
-::: callout TLDR
+## Desktop Implementations
 
-Looking for an easy and opinionated way to get started with IPFS?
+Looking for an easy way to get started? Install these tools for no-code access to the IPFS Amino Public Network.
 
-**User-Friendly Options**
-  - [IPFS Desktop](../install/ipfs-desktop.md), an user-friendly GUI application that bundles an IPFS Kubo node for self-hosting, file manager, peer manager, and content explorer.
-  - [IPFS Companion](../install/ipfs-companion.md), a browser extension that compliments IPFS Desktop, loads compatible websites and file paths from a local IPFS Kubo node.
+| Name           | URL                                   | Language(s)    | What it's trying to do                                                                        |
+|----------------|---------------------------------------|----------------|-----------------------------------------------------------------------------------------------|
+| IPFS Desktop   | <https://github.com/ipfs/ipfs-desktop>| javascript    | Desktop application bundling a Kubo node with file manager, peer manager and content explorer  |
+| IPFS Companion | <https://github.com/ipfs/ipfs-companion>| javascript  | Browser extension that runs a local Kubo node and gets content from the public network         |
 
-**Command-Line & Infrastructure Tools**
-  - [Kubo](../install/command-line.md), all-in-one demon service with command-line and HTTP RPC interface for power users who want to self-host, but don't need GUI.
-  - [IPFS Cluster](../install/server-infrastructure.md), data orchestration across a swarm of IPFS Kubo daemons by allocating, replicating and tracking a global pinset distributed among multiple peers.
-  - [Rainbow](https://github.com/ipfs/rainbow/#readme), production-grade standalone IPFS HTTP [Gateway](../concepts/glossary.md#gateway) service powering [public gateways](../concepts/public-utilities.md#public-ipfs-gateways).
-:::
+## Popular Node Implementations and Tools
 
-## Popular or Actively Maintained
+For developers and operators.
```

Comment:
I wonder if it makes sense to add an "IPFS Mainnet" column to indicate whether an implementation supports it.

Reply:
Let's address in a separate PR. I think it would make sense as a bigger matrix of features/support, but maybe not as the only one.

Reply:
Yep. It's actually a matrix we need rather than Mainnet with the specifics